### PR TITLE
feat: create sign-up page

### PR DESCRIPTION
**Pull request by jaydelim**
# Description
Created a sign-up page. This page is re-using the `NavBar` component. This page also includes a few user-input components: `UserName`, `Email` and `Password`.

# Testing Instruction
Run `npm start`
Visit `http://localhost:3000/`

# Screenshots
## Desktop
Screen Shot 2022-08-04 at 4 49 10 PM

## Tablet
Screen Shot 2022-08-04 at 4 49 24 PM

## Mobile
Screen Shot 2022-08-04 at 4 49 33 PM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import Homepage from "./pages/Homepage";
+import SignUpPage from "./pages/SignUpPage";
 
 function App() {
   return (
-    <Homepage/>
+    // <Homepage/>
```
Comment (jaydelim) [inline src/App.js:7]: As the instruction of the sign-up page was to create a non-functional (UI only) page, I've commented out `Homepage` from App.js file. Please let me know if i need to use React Router instead. Thanks!

File: src/components/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return(
+    <div>
+      <div className="flex flex-col items-center justify-center py-4 pt-12">
+        <p className="text-5xl font-light">Sign Up</p>
+        <p className="text-[#5cb85c] pt-2 text-lg my-2">Have an account?</p>
+      </div>
+
+      <div className= "flex items-center justify-center px-4">
+        <div className="flex flex-col w-[640px] gap-y-5">
+          <UserName />
```
Comment (YoussefLaunchUp) [inline src/components/SignUp.js]: All these components look pretty similar inside. Lets merge them all into 1 component that accepts parameters to change how it works.

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Lets try to make the components more reusable for more use cases.

File: src/components/SignUp.js (hunk 2 above)
Comment (YoussefLaunchUp) [reply src/components/SignUp.js]: Referring to Username, email and password.

Comment (jaydelim) [reply src/components/SignUp.js]: Addressed. `Username`, `Email`, `Password` components have been combined as a re-usuable 'InputForm' component.  Please see my updated codes. Thanks!

File: src/components/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  return(
+    <div>
+      <div className="flex flex-col items-center justify-center py-4 pt-12">
+        <p className="text-5xl font-light">Sign Up</p>
+        <p className="text-[#5cb85c] pt-2 text-lg my-2">Have an account?</p>
+      </div>
+
+      <div className="flex items-center justify-center px-4">
+        <div className="flex flex-col w-[640px] gap-y-5">
+          <InputForm fields={["Username","Email","Password"]} />
```
Comment (YoussefLaunchUp) [inline src/components/SignUp.js]: The Input form should only make 1 input box each. So you can use InputForm 3 times here with a different field for each.

File: src/components/Form.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    <div className="border-slate-300 border rounded-md h-16 flex items-center p-4 text-xl font-light text-stone-400">
+      <input type="text" name={props.info} placeholder={props.info} className="focus:outline-none" />
+    </div>
+  )
+}
+
+function InputForm (props) {
+  return (
+    <div className="flex flex-col w-full gap-y-5">
+      {props.fields.map(field => (<FormBox info={field}/>))}
```
Comment (YoussefLaunchUp) [inline src/components/Form.js]: No need for FormBox to be different from InputForm. You can merge both.

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Great work, some small changes needed.

File: src/components/Form.js (hunk 4 above)
Comment (jaydelim) [reply src/components/Form.js]: Addressed. `FormBox` has been merged into `InputForm`, and has been removed.

File: src/components/SignUp.js (hunk 3 above)
Comment (jaydelim) [reply src/components/SignUp.js]: Addressed. `InputForm` has been updated to show one field at a time. Thanks!

**Review by YoussefLaunchUp — APPROVED** [review]

**Comment by YoussefLaunchUp** [comment]
Great work.

**Pull request merged**